style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  height: auto;
+  background-color: #210b54;
+}
```

Comment:
- [ ] For these sections, focus  on the the margins (top, bottom, left, right) defined in the design
- [ ] Your page
![Screenshot from 2021-09-17 12-24-14](https://user-images.githubusercontent.com/65501535/133759815-feca10a8-6a69-4c34-af1a-076731ce40e5.png)

Original Page.
![Screenshot from 2021-09-17 12-25-04](https://user-images.githubusercontent.com/65501535/133759947-80f8f165-57a0-4863-9ff1-902ca7a44d96.png)

Reply:
Thank you.